modin/pandas/test/test_io.py

```diff
@@ -490,12 +497,33 @@ def test_read_csv_mangle_dupe_cols(self):
             pytest.xfail(
                 "processing of duplicated columns in HDK storage format is not supported yet - issue #3080"
             )
-        with ensure_clean() as unique_filename:
+        with ensure_clean() as unique_filename, (
+            pytest.warns(
+                FutureWarning, match="'mangle_dupe_cols' keyword is deprecated"
+            )
+            if PandasCompatVersion.CURRENT == PandasCompatVersion.LATEST
```

Comment:
Shouldn't we be checking if `PandasCompatVersion.CURRENT > 1.4.0`? I guess what we have now is fine too, but maybe it's better to be explicit about when things got deprecated?

Reply:
The version can only be `PY36` or `LATEST`. I do think we should check pandas version rather than compatibility version, so I made the code check pandas version instead.